contrib/test/integration/build/kubernetes.yml

```diff
@@ -13,6 +13,11 @@
   args:
     chdir: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
 
+- name: build kubernetes
```

Comment:
Is this meant to be a duplicate of what we have at line 21 below?

Reply:
copy paste error :\
supposed to be 'clean kubernetes'

Reply:
can we get this PR though and I'll open a new PR fixing this nit? openshift-robot opened this one

Reply:
sure, sounds good.